evil-commands.el

```diff
@@ -4710,6 +4710,18 @@ and redisplay the current buffer there."
   :repeat nil
   (evil-move-window 'below))
 
+;;; Tab commands
+
+(evil-define-command evil-tab-next (&optional arg)
```

Comment:
Needs the `:repeat nil` property.

(That ought to be added to the other `tab-bar` functions as well in `evil-integration.el`, however that is a separate issue.)